atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1484,6 +1486,32 @@ public void commitThrowsIfRolledBackAtCommitTime_expiredLocks() {
                 .hasLocksExpired(1);
     }
 
+    @Test
+    public void commitThrowsIfLockAcquisitionFails() {
+        final Cell cell = Cell.create(PtBytes.toBytes("row1"), PtBytes.toBytes("column1"));
```

Comment:
As you've probably noticed, this class can be quite self-inconsistent in how tests are written. 
However, there's a few things we can do to make this new addition more legible.

Here, I think we can just use the existing TEST_CELL constant